app/src/main/java/com/example/vmedvediev/redditapp/MainActivity.kt

```diff
@@ -111,4 +85,15 @@ class MainActivity : AppCompatActivity() {
             }
         })
     }
-}
+
+    private fun getRetrofit() : FeedAPI {
```

Comment:
This function initiates Retrofit instance, so it must be `initRetrofit()` or `setupRetrofit()` (instead of `getRetrofit()` )